package src.day44_Exceptions;

import java.util.ArrayList;
import java.util.Arrays;

public class try_catchBlocks2 {
    public static void main(String[] args) {
        System.out.println("Hello");

        try {

            Thread.sleep(3000);

        }catch(Exception  e){

        }


        System.out.println("World");

    }

}
